shop/src/components/bottom-sheet-option/index.json

```
@@ -0,0 +1,7 @@
+{
+  "component": true,
+  "usingComponents": {
+    "bottom-sheet": "@tiki.vn/tini-ui/es/bottom-sheet/index",
+    "card-select": "/components/card-select/index"
```

```suggestion
    "card-select": "components/card-select/index"
```

Không cần / cũng đc nhé e

---

Em đã update